docs/tutorials/services/constrain-motion.md

```diff
@@ -0,0 +1,433 @@
+---
+title: "Add Constraints and Transforms to a Motion Plan"
+linkTitle: "Add Motion Constraints"
+weight: 25
+type: "docs"
+description: "Use constraints and transforms with the Motion Service."
+webmSrc: "/tutorials/videos/motion_constraints.webm"
+mp4Src: "/tutorials/videos/motion_constraints.mp4"
+images: ["/tutorials/videos/motion_constraints.gif"]
+videoAlt: "An arm moving a cup from one side of a tissue box to the other, across a table. The cup stays upright."
+tags: ["arm", "gripper", "motion", "services"]
+# SMEs: Motion Team
+---
+
+{{< alert title="Caution" color="caution" >}}
+Be careful when instructing robot arms to move.
+Before running any code, ensure your robotic arm has enough space and that there are no obstacles.
+Also pay attention to your surroundings, double-check your code for correctness, and make sure anyone nearby is aware and alert before issuing commands to your robot.
+{{< /alert >}}
+
+{{<gif webm_src="../../videos/motion_constraints.webm" mp4_src="../../videos/motion_constraints.mp4" poster="../../videos/motion_constraints.jpg" alt="An arm moving a cup from one side of a tissue box to the other, across a table. The cup stays upright." class="alignright" max-width="250px">}}
+
+Say you want your robot to pass you a cup of tea, but you don't want it to spill the water or bump into other objects on the table.
+
+If you followed along in [part 2 of the Motion Service tutorial series](../plan-motion-with-arm-gripper/), you used the [Motion Service](/services/motion/) to move a robot arm and end effector to desired positions.
```

Comment:
```suggestion
If you followed along with the [Plan Motion with an Arm tutorial](../plan-motion-with-arm-gripper/), you used the [Motion Service](/services/motion/) to move a robot arm and end effector to desired positions.
```

^ This adds more useful info. We don't otherwise refer to this as a series currently - so I think this is better until we decide on broader changes.